project/src/main/webapp/event-edit.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html> 
+  <head> 
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="create-event.css" />
+    <script src="script.js"></script>
+    <script src="event-edit.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js"></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Public+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Bootstrap CDN links -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body onload = "getEventDetails()">
+    <div id="header"></div>
+    <br /><br />
+    <h3>Edit Your Event</h3>
+    <br />
+    <form id="create-event" action="/create-event" method="POST">
+      <div class="col">
+        <div class="input-group">
+          <div class="input-group-prepend">
+            <span class="input-group-text" id="basic-addon1">Name</span>
+          </div>
+          <input
+            type="text"
+            class="form-control"
+            placeholder="Event Name"
+            id="name"
+            name="name"
+            aria-describedby="basic-addon1"
+          />
+        </div>
+        <br />
+        <div class="row">
+          <div class="col-lg-6">
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text" id="basic-addon1">Date</span>
+              </div>
+              <input
+                type="text"
+                class="form-control"
+                placeholder="dd/MM/yyyy"
+                id="date"
+                name="date"
+                aria-describedby="basic-addon1"
+              />
+            </div>
+          </div>
+          <br />
+          <br />
+          <div class="col-lg-6">
+            <div class="input-group">
+              <div class="input-group-prepend">
+                <span class="input-group-text" id="basic-addon1">Time</span>
+              </div>
+              <input
+                type="text"
+                class="form-control"
+                placeholder="Time"
+                id="time"
+                name="time"
+                aria-describedby="basic-addon1"
+              />
+            </div>
+          </div>
+        </div>
+        <br />
+        <div class="inline">
+          <text> Tags: </text>
+          <div class="container">
+            <span class="badge badge-primary text-medium"> Tag 1</span>
```

Comment:
This part at least looks like an older version - newer version of event edit should include the updated interests and skills section

Reply:
ty for looking so carefully! will fix

Reply:
Could you link me to a PR / file where this is done?

Reply:
You can look at create-event.html for examples of how it looks like rn